tests/PhpCollection/Tests/LRUMapTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace PhpCollection\Tests;
+
+use PhpCollection\LRUMap;
+use PhpCollection\Map;
+use PhpCollection\Tests\MapTest;
+
+class LRUMapTest extends MapTest
```

Comment:
why extending from MapTest ? This will run the MapTest twice, which does not make sense